lib/cleanup.js

```diff
@@ -2,8 +2,16 @@ function removeLeadingZerosFromStreet(token) {
   return token.replace(/^(?:0*)([1-9]\d*(st|nd|rd|th))/,'$1');
 }
 
+// expand '-str.' to '-strasse'
+// note: '-straße' is only used in Germany, other countries like
+// switzerland use 'strasse'.
+function expandGermanicStreetSuffixes(token) {
```

Comment:
is this functionality still required? it seems to be a duplicate of the functions in `lib/streams/germanAbbreviationStream.js`